doc/architecture/frontend.md

```diff
@@ -0,0 +1,87 @@
+# Frontend Architecture
+
+## Structure
+
+- `__mocks__/`: Contains mocked modules for tests.
+- `assets/`: Contains small asset files like SVGs.
+- `bindings/`: Contains the generated API bindings.
+- `components/`: Contains components that are shared across features.
+- `config/`: Contains initialization code for the environment and libraries.
+- `features/`: Contains the distinct features of the application.
+- `generated/`: Contains generated code, for now only docs.
+- `hooks/`: Contains hooks that are shared across features.
+- `icons/`: Contains svg icons.
```

Comment:
I think the icons could be moved to assets

Reply:
Yes, I created a separate issue for this #355